nextcloudappstore/core/api/v1/release/info.xsd

```diff
@@ -245,7 +245,7 @@
     <!-- first level elements -->
     <xs:simpleType name="id">
         <xs:restriction base="limited-string">
-            <xs:pattern value="[a-z]+[a-z_]*[a-z]+"/>
+            <xs:pattern value="[a-z]+[a-z0-9_]*[a-z0-9]+"/>
```

Comment:
Good point, also thought about this but should not be an issue. Sure you can do something like that but that would also be a valid app id. It's probably better to not restrict it harder than needed especially because app authors need to read the regex if it fails.